packages/components/TabList/src/TabList/useTabList.ts

```diff
@@ -72,6 +73,49 @@ export const useTabList = (props: TabListProps): TabListInfo => {
     [setTabKeys],
   );
 
+  const incrementTabKey = React.useCallback(
+    (increment: number) => {
+      if (increment === 0) {
+        return;
+      }
+
+      const currentIndex = tabKeys.indexOf(selectedTabKey);
+
+      if (currentIndex === -1 && __DEV__) {
+        console.error(`Selected tab key does not exist in tab key array. Tab Key: ${selectedTabKey}. Keys: ${tabKeys}`);
+      }
+
+      // We want to only switch selection to non-disabled tabs. This skips over disabled ones.
+      const direction = increment > 0 ? 1 : -1;
+      increment = increment / direction; // abs
```

Comment:
can increment be its own variable so that we're not reusing the param passed in?

Reply:
This function seems over-complicated by trying handle any amount of increment. My understanding is that the only use case right now is 1 or -1, so in order for simplicity I think it would be best to start with just handling those two values.

Reply:
No that's fair criticism. Method has been changed to be simpler.